cernopendata/modules/fixtures/data/records/cms-simulated-datasets-2012.json

```diff
@@ -48839,5 +48839,144 @@
     "validation": {
       "description": "The generation and simulation of simulated Monte Carlo data has been validated through general CMS validation procedures."
     }
+  },
+  {
+    "abstract": {
+      "description": "<p>Simulated dataset ADDdiLepton_LambdaT-1600_Tune4C_8TeV-pythia8 in AODSIM format for 2012 collision data.</p><p>See the description of the simulated dataset names in: <a href=\"/about/CMS-Simulated-Dataset-Names\">About CMS simulated dataset names</a>.</p>"
+    },
+    "accelerator": "CERN-LHC",
+    "categories": {
+      "primary": "Exotica",
+      "secondary": [
+        "Extra Dimensions"
+      ],
+      "source": "CMS Collaboration"
+    },
+    "collaboration": {
+      "name": "CMS Collaboration",
+      "recid": "453"
+    },
+    "collections": [
+      "CMS-Simulated-Datasets"
+    ],
+    "collision_information": {
+      "energy": "8TeV",
+      "type": "pp"
+    },
+    "date_created": [
+      "2012"
+    ],
+    "date_published": "2018",
+    "date_reprocessed": "2012",
+    "distribution": {
+      "availability": "ondemand",
+      "formats": [
+        "aodsim",
+        "root"
+      ],
+      "number_events": 200000,
+      "number_files": 29,
+      "size": 73407378326
+    },
+    "experiment": "CMS",
+    "license": {
+      "attribution": "CC0"
+    },
+    "methodology": {
+      "description": "<p>These data were generated in several steps:</p><p><strong>Note</strong><br>\n                              To get the exact LHE and generator's parameters, see <a href=\"/docs/cms-mc-production-overview\">CMS Monte Carlo production Overview</a></p>\n                           ",
```

Comment:
Also, the leading phrase is the old one:

```
To get the exact LHE and generator's parameters, see CMS Monte Carlo production Overview
```

while we have amended it for 2012 records to something like:

```
These data were generated in several steps (see also CMS Monte Carlo Production Overview):
```

Reply:
I'll fix those and repush and merge so that @pamfilos can start working on he search topics... #2563